hack/arktos-cni.rc

```diff
@@ -66,6 +69,23 @@ install_cni_bridge() {
     echo "done with bridge cni plugin installation"
 }
 
+install_cni_mizar() {
+    echo "Ensuring firewall to allow traffic forward by default"
+    sudo iptables -S FORWARD | grep '\-P' | grep DROP && sudo iptables -P FORWARD ACCEPT
```

Comment:
Why do we need this? Does flannel need this? If not, this is a security hole.

Reply:
We have reused this code from the install_cni_bridge.
This rule prevents conflicts between docker iptables rules and pod communication in [Kata Containers and containerd](https://github.com/kata-containers/documentation/blob/master/how-to/how-to-use-k8s-with-cri-containerd-and-kata.md#install-and-configure-containerd)

Reply:
Why doesn't the upstream local-up-cluster need it? Also the issue referenced is from 2017 and applies to docker 1.13. Is it still an issue?

Reply:
We have retested this by removing the code and it is working fine.

Reply:
Why is the iptables code still here?

Reply:
We have removed the code in the recent commit, thanks.